mobilebot/mobilebot.c

```diff
@@ -145,8 +146,8 @@ void read_mb_sensors(){
     }
 
     // Read encoders    
```

Comment:
Are updating left and right wheel velocities (based on encoder_delta) part of this PR?

Reply:
that's for a later implementation